SYCL/ESIMD/lsc/Inputs/lsc_usm_store.hpp

```diff
@@ -82,7 +83,8 @@ bool test(uint32_t pmask = 0xffffffff) {
 
             if constexpr (transpose) {
               simd<T, VS> vals(new_val + elem_off, 1);
-              lsc_block_store<T, VS, DS, L1H, L3H>(out + elem_off, vals);
+              lsc_block_store<T, VS, DS, L1H, L3H>(out + elem_off, vals,
+                                                   Flags{});
```

Comment:
Please verify both variants: with and without Flags parameter. For example:
```suggestion
              if constexpr (sizeof(T) < 4)
                  lsc_block_store<T, VS, DS, L1H, L3H>(out + elem_off, vals,
                                                       Flags{});
              else
                  lsc_block_store<T, VS, DS, L1H, L3H>(out + elem_off, vals);
```